p3/utils.py

```diff
@@ -318,7 +318,7 @@ def group_all_talks_by_admin_type(conference, talk_status='accepted'):
         with status `talk_status` separated by administrative type. """
         # Group by admin types
         talks = OrderedDict()
-        for adm_type, type_name in dict(cmodels.TALK_ADMIN_TYPE).items():
+        for adm_type, type_name in list(dict(cmodels.TALK_ADMIN_TYPE).items()):
```

Comment:
Unnecessary `list`.

Reply:
For future reference: find . -type f -name \*.py -exec gawk -i inplace '{print gensub(/for (.+) in list\((.+)\):/, "for \\1 in \\2:", "g");}' {} \;